e2e/scripts/st_set_page_config_icon.py

```diff
@@ -0,0 +1,31 @@
+import streamlit as st
+import os, fnmatch
+
+
+def is_assets_favicon_path(path):
```

Comment:
Haha this code got complicated, I would just add a comment on the reason why this is happening (we should note that this later can be a e2e_test_util library